packages/world_countries/example/android/app/build.gradle.kts

```
@@ -39,6 +37,128 @@ android {
 }
+
+/* ------- Helper function to run shell commands ------- */
+fun String.runCommand(workingDir: File = File(".")): String? = try {
+    ProcessBuilder(*split(" ").toTypedArray())
+        .directory(workingDir)
+        .redirectOutput(ProcessBuilder.Redirect.PIPE)
+        .redirectError(ProcessBuilder.Redirect.PIPE)
+        .start()
+        .apply { waitFor(10, TimeUnit.SECONDS) }
+        .inputStream.bufferedReader().readText().trim()
+} catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)